plugins/foreman_remote_execution/1.3/index.md

```diff
@@ -0,0 +1,877 @@
+---
+layout: plugin
+title: Foreman Remote Execution 1.3 Manual
+version: 1.3
+---
+
+# 1. {{ page.title }}
+
+This plugin enables Foreman to run arbitrary commands on hosts using different
+providers. Initially only an SSH provider is supported but we plan to add more.
+Communication goes through the smart proxy so Foreman does not have to have
+direct access to the target host and can scale to control many hosts. A command
+can be customized similarly to provisioning templates or partition tables. A
+number of templates are included for running basic commands, installing
+packages, etc. For more details see the
+[JobTemplates](plugins/foreman_remote_execution/{{page.version}}/index.html#3.1JobTemplates)
+section.
+
+Below is a short demo of running a single command on a particular host, with
+the ability to view the output live:
+
+![GIF Demo](/plugins/foreman_remote_execution/{{page.version}}/remote_execution.gif)
+![WEBM Demo](/plugins/foreman_remote_execution/{{page.version}}/output.gif)
+
+The command can be executed on multiple servers at once while specific parts of
+it can vary per host - in other words you can use variables in your commands.
+Variable values can be filled by given host fact, smart class parameter, smart
+variable or even host parameter. Last but not least, like the "command" example
+above, the value can be specified by user when they trigger the execution.
+
+A few examples of tasks that this plugin helps you achieve:
+
+- Install/update/remove a package
+- Bootstrap a configuration management agent
+- Trigger a Puppet/Salt/Chef run
+
+## 1.1 Components
+
+The whole remote execution system is pluggable, making it easy to add more
+providers of communication later. Therefore it's split into several components
+that must be installed on Foreman and the Smart Proxy.
+
+Due to various reasons, dynflow was separated from Smart Proxy into a standalone
+service called smart_proxy_dynflow_core. Most of the gems originally required for
+remote execution also followed this example and were split into regular and -core part.
+The original gems are usually either Foreman or Smart Proxy plugins, the -core parts
+are required by the smart proxy dynflow core service.
+
+This split approach also allows us to simplify the deployment by loading the -core gems
+into Smart Proxy or even directly into Foreman and thus lowering the entry barrier
+at the cost of lower performance.
+
+Please note that on Debian Dynflow will be set up as part of the Foreman Proxy. On other
+platforms it will be a standalone service.
```

Comment:
comma missing after "Debian"

Reply:
Also, isn't EL7 the only only left of "other platforms"? If yes, that whole paragraph should be reworded.